### PR TITLE
Aae 18513 update JDK to 21 to fix the build

### PR DESCRIPTION
**Pull request by wojciech-piotrowiak**

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
-  nextVersion: 7.7.0-SNAPSHOT
+  version: 8.2.0
+  nextVersion: 8.3.0-SNAPSHOT
+  notesStartTag: 8.1.0
   mock: false
+  baseTag:
+    activiti: 8.2.0-alpha.25
+    activitiCloud: 8.2.0-alpha.35
+    commonChart: 8.2.0-alpha.4
+    fullChart: 8.2.0-alpha.44
```
Comment (erdemedeiros) [inline release.yaml]: IMO, it's better to not include this latest part (`baseTag`) that's automatically generated when the rc version is created

**Review by erdemedeiros — COMMENTED** [review]
I added a small comment, LGTM otherwise

File: release.yaml (hunk 1 above)
Comment (wojciech-piotrowiak) [reply release.yaml]: thanks, fixed

**Review by gicappa — APPROVED** [review]
LGTM

**Pull request merged**